src/ConfigurationFactory.php

```diff
@@ -135,6 +135,12 @@ protected function createWithConfig(ContainerInterface $container, string $confi
             $configuration->setEntityListenerResolver($config['entity_listener_resolver']);
         }
 
+        if (is_string($config['schema_assets_filter'])) {
+            $configuration->setSchemaAssetsFilter($container->get($config['schema_assets_filter']));
+        } elseif ($config['schema_assets_filter'] !== null) {
+            $configuration->setSchemaAssetsFilter($config['schema_assets_filter']);
```

Comment:
This execution branch is IMO to be dropped

Reply:
ok, but in my case its enough to predefined list instead of some analog of symfony compiler pass